contributors/devel/conformance-tests.md

```diff
@@ -1,57 +1,69 @@
-linux kernel or for a kubelet to run (i.e. git)
+Conformance tests should be boring. They should rely on nothing special, test only
+features that are available everywhere, and generate the same results every time.
+
+More specifically, a test is eligble for promotion to conformance if:
+
+- it tests only GA, non-optional features or APIs (ie: no alpha or beta endpoints, 
+  no feature flags required)
+- it works for all providers (ie: no `SkipIfProviderIs`/`SkipUnlessProviderIs` calls)
+- it is non-privileged (ie: no root on nodes, network, nor cluster)
+- it works without access to the public internet
+- it works without non-standard filesystem permissions granted to containers or users
+  (eg: can't share a writable host /tmp with a container)
+- it does not rely on any binaries that would be required for the linux kernel or
+  kubelet to run (ie: can't rely on git)
+- it is stable and runs consistently (ie: no flakes)
+
+Examples of features which are not eligible for conformance tests:
+
+- node/platform-reliant features, eg: multiple disk mounts, GPUs, high density, etc.
+- optional features, eg: policy enforcement
+- cloud-provider-specific features, eg: GCE monitoring, S3 Bucketing, etc.
+- anything that requires a non-default admission plugin
 
 ### Conformance Test Version Skew Policy
 
 As each new release of Kubernetes provides new functionality, the subset of
 tests necessary to demonstrate conformance grows with each release. Conformance
 is thus considered versioned, with the same backwards compatibility guarantees
 as laid out in [our versioning policy](/contributors/design-proposals/release/versioning.md#supported-releases-and-component-skew).
-Conformance tests for a given version should be run off of the release branch
-that corresponds to that version. Thus `v1.2` conformance tests would be run
-from the head of the `release-1.2` branch. eg:
 
- - A v1.3 development cluster should pass v1.1, v1.2 conformance tests
+To quote:
+
+> For example, a v1.3 master should work with v1.1, v1.2, and v1.3 nodes, and 
+> should work with v1.2, v1.3, and v1.4 clients.
 
- - A v1.2 cluster should pass v1.1, v1.2 conformance tests
+Conformance tests for a given version should be run off of the release branch
+that corresponds to that version. Thus `v1.2` conformance tests would be run
+from the head of the `release-1.2` branch.
 
- - A v1.1 cluster should pass v1.0, v1.1 conformance tests, and fail v1.2
-conformance tests
+For example, suppose we're in the midst of developing kubernetes v1.3. The
+following clusters must pass conformance tests built from the following branches:
 
+| cluster version | master | release-1.3 | release-1.2 | release-1.1 |
+| --------------- | -----  | ----------- | ----------- | ----------- |
+| v1.3.0-alpha    | yes    | yes         | yes         | no          |
+| v1.2.x          | no     | no          | yes         | yes         |
+| v1.1.x          | no     | no          | no          | yes         |
```

Comment:
@tpepper Yes, it would be reasonable to impose an earlier deadline. We continue to have the problem that new features are pushed through late in the process, are found to be buggy, and then are fixed in subsequent patch releases.